projects/js-packages/charts/src/charts/sparkline/sparkline.module.scss

```diff
@@ -0,0 +1,22 @@
+.sparkline {
+	display: inline-block;
+	vertical-align: middle;
+	line-height: 1;
+
+	svg {
+		display: block;
+		overflow: visible;
+	}
+
+	path {
+		vector-effect: non-scaling-stroke;
+	}
+
+	&--empty {
+		display: inline-block;
+	}
+
+	&--single-point {
+		display: block;
```

Comment:
The SVG for single point already has display: block via the parent's .sparkline svg rule?